cekit/descriptor/osbs.py

```diff
@@ -20,3 +28,27 @@ class Osbs(Descriptor):
     def __init__(self, descriptor):
         self.schemas = osbs_schema
         super(Osbs, self).__init__(descriptor)
+
+        if 'configuration' in self:
+            self['configuration'] = Configuration(self['configuration'])
+
+
+class Configuration(Descriptor):
+    """Internal object represeting OSBS configuration subObject
+
+    Args:
+      descriptor - yaml contianing OSBS configuration"""
+
+    def __init__(self, descriptor):
+        self.schemas = configuration_schema
+        super(Configuration, self).__init__(descriptor)
+        self.skip_merging = ['container', 'container_file']
+
+    def _prepare_configuration(self):
+        if 'container' in self and 'container_file' in self:
+            raise CekitError('You cannot specifu container and container_file together!')
+
+        if 'container_file' in self:
+            with open(self['container_file'], 'r') as file_:
```

Comment:
Don't we want some check if the file exists?

Reply:
check is there in code above now. So this should be ok